src/packer.pkr.hcl

```diff
@@ -83,51 +102,60 @@ source "amazon-ebs" "example" {
   region             = var.build_region
   region_kms_key_ids = var.region_kms_keys
   skip_create_ami    = var.skip_create_ami
-  source_ami         = data.amazon-ami.debian_bullseye.id
-  ssh_username       = "admin"
+  source_ami         = data.amazon-ami.windows_server_2022.id
   subnet_filter {
     filters = {
       "tag:Name" = "AMI Build"
     }
   }
   tags = {
-    Application        = "Example"
-    Base_AMI_Name      = data.amazon-ami.debian_bullseye.name
+    Application        = "Windows Server 2022"
```

Comment:
Do we want to update this?

Reply:
Addressed here https://github.com/cisagov/windows-commando-vm-packer/pull/20/commits/fca2728ac8fdb8298991237ff2c1d66dc921be17#diff-61505b29b2bfc888d227eb40217a3c6d44ec3ba4dc88b44bdf89a7fa63d3358fR112